package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "standard",
     "lint:fix": "standard --fix",
     "redis": "docker run -p 6379:6379 --rm redis",
-    "test": "standard && tap test/*.test.js && npm run typescript",
+    "test": "standard && tap test/*.test.js test/**/*.test.js && npm run typescript",
```

Comment:
```suggestion
    "test": "standard && tap \"test/**/*.test.js\" && npm run typescript",
```

Reply:
Yes it works as intended when escaping the quotes. Tested with Archlinux, Windows 10 & 11 (I can test with MacOS when I get back home).

What's your working environment ?

NB: the `not-found-handler-rate-limited.js` test file is currently not run. It needs to be renamed to something like `not-found-handler-rate-limited.test.js`

Reply:
> Yes it works as intended when escaping the quotes. Tested with Archlinux, Windows 10 & 11 (I can test with MacOS when I get back home).
> 
> 
> 
> What's your working environment ?

Mac OS 😅

> NB: the `not-found-handler-rate-limited.js` test file is currently not run. It needs to be renamed to something like `not-found-handler-rate-limited.test.js`

Weird I didn't touch that file

Reply:
https://github.com/darkgl0w/fastify-rate-limit/runs/5188076153?check_suite_focus=true
Triggered a CI run on Mac OS, it works as expected with the escaped quotes. :P

Reply:
Great! Thanks!